library/core/src/wtf8.rs

```diff
@@ -1046,21 +572,19 @@ impl Iterator for EncodeWide<'_> {
     }
 }
 
+impl fmt::Debug for EncodeWide<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("EncodeWide").finish_non_exhaustive()
```

Comment:
Unlike `Wtf8CodePoints`, it's not entirely clear how you'd reconstruct the original string from `EncodeWide` and include the unpaired surrogate, so, I decided to not bother for now. Someone can write a better debug implementation later.